webapp/app/players/controllers/playerDetail.controller.js

```diff
@@ -119,18 +119,16 @@ angular.module('app.players').controller("PlayerDetailController", function($sco
             failureCallback);
     };
 
-    $scope.getMergePlayers = function(viewValue) {
-        players = $scope.playerService.getPlayerListFromQuery(viewValue,
-            function(player) {return player.id != $scope.playerId});
-        return players;
-    }
+    $scope.notCurrentPlayer = function(player) {
```

Comment:
kind of janky (took from https://stackoverflow.com/questions/11753321/passing-arguments-to-angularjs-filters).

I would've liked to

- pass `'!{{player.id}}'` here (https://github.com/ripgarpr/garpr/pull/191/files#diff-a7b63b0f956f2fbcec83408cc2b3749cR74) 
- use `@` here (https://github.com/ripgarpr/garpr/pull/191/files#diff-c5fa04dc2c1bb06c85e1392dce811603R11) 
- do something like `| filter:{\'{{filteredOut}}\'}` here (https://github.com/ripgarpr/garpr/pull/191/files#diff-f1739b2348842da30e79e16736adcf07R6)

but it just doesn't work :(